extensions/arc/src/ui/dashboards/controller/controllerDashboardOverviewPage.ts

```diff
@@ -0,0 +1,193 @@
+			this.eventuallyRunOnInitialized(() => {
+				this.handleRegistrationsUpdated();
+			});
+		});
+		this.refresh().catch(e => {
+			console.log(e);
+		});
+	}
+
+	public get title(): string {
+		return loc.overview;
+	}
+
+	public get id(): string {
+		return 'controller-overview';
+	}
+
+	public get icon(): { dark: string, light: string } {
+		return IconPathHelper.properties;
+	}
+
+	protected async refresh(): Promise<void> {
+		await this._controllerModel.refresh();
+	}
+
+	public get container(): azdata.Component {
+
+		const rootContainer = this.modelView.modelBuilder.flexContainer()
+			.withLayout({ flexFlow: 'column' })
+			.component();
+
+		const contentContainer = this.modelView.modelBuilder.divContainer().component();
+		rootContainer.addItem(contentContainer, { CSSStyles: { 'margin': '10px 20px 0px 20px' } });
+
+		this._propertiesContainer = this.modelView.modelBuilder.propertiesContainer().component();
+
+		contentContainer.addItem(this._propertiesContainer);
+
+		const arcResourcesTitle = this.modelView.modelBuilder.text()
+			.withProperties<azdata.TextComponentProperties>({ value: loc.arcResources })
+			.component();
+
+		contentContainer.addItem(arcResourcesTitle, {
+			CSSStyles: cssStyles.title
+		});
+
+		this._arcResourcesTable = this.modelView.modelBuilder.declarativeTable().withProperties<azdata.DeclarativeTableProperties>({
+			data: [],
+			columns: [
+				{
+					displayName: loc.name,
+					valueType: azdata.DeclarativeDataType.string,
+					width: '33%',
+					isReadOnly: true,
+					headerCssStyles: cssStyles.tableHeader,
+					rowCssStyles: cssStyles.tableRow
+				}, {
+					displayName: loc.type,
+					valueType: azdata.DeclarativeDataType.string,
+					width: '33%',
+					isReadOnly: true,
+					headerCssStyles: cssStyles.tableHeader,
+					rowCssStyles: cssStyles.tableRow
+				}, {
+					displayName: loc.computeAndStorage,
+					valueType: azdata.DeclarativeDataType.string,
+					width: '34%',
+					isReadOnly: true,
+					headerCssStyles: cssStyles.tableHeader,
+					rowCssStyles: cssStyles.tableRow
+				}
+			],
+			width: '100%',
+			ariaLabel: loc.arcResources
+		}).component();
+
+		const arcResourcesTableContainer = this.modelView.modelBuilder.divContainer()
+			.withItems([this._arcResourcesTable])
+			.component();
+
+		contentContainer.addItem(arcResourcesTableContainer);
+		this.initialized = true;
+		return rootContainer;
+	}
+
+	public get toolbarContainer(): azdata.ToolbarContainer {
+
+		const createNewButton = this.modelView.modelBuilder.button().withProperties<azdata.ButtonProperties>({
+			label: loc.createNew,
+			iconPath: IconPathHelper.add
+		}).component();
+
+		const deleteButton = this.modelView.modelBuilder.button().withProperties<azdata.ButtonProperties>({
+			label: loc.deleteText,
+			iconPath: IconPathHelper.delete
+		}).component();
+
+		const resetPasswordButton = this.modelView.modelBuilder.button().withProperties<azdata.ButtonProperties>({
```

Comment:
Changing the controller's password after deployment is not currently supported.  Was that in the design?

Reply:
Yeah it was - I'll remove this then.